Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Header.css';
+import ImageThumbnail from './ImageThumbnail.js';
+const HeaderContainer = () => {
```

Comment:
just a short suggestion is that you could just have your function output a render function without using a return, like so. const HeaderContainer = () = ( <ImageThumbnail /> ). It would definitely make you're code shorter and cleaner that way